docs/source/installation.rst

```diff
@@ -30,8 +30,9 @@ Generic requirements
   with bazaar repositories.
 * ``pyuv`` python package. Required for :ref:`libuv-based watcher 
   <config-common-watcher>` to work.
-* ``i3-py``, `available on github <https://github.com/ziberna/i3-py>`_. Required 
-  for i3wm bindings and segments.
+* ``i3-ipc`` python package. Required for i3wm bindings and segments.
+* ``xrandr`` program. Required for the multi-monitor lemonbar binding and the 
+  :func:`i3wm output_lister <powerline.listers.i3wm.output_lister>`.
```

Comment:
I tried it locally and it did, but I can match that style.

It looked a bit weird though, "output_lister" is written in monospaced font with a well around it, it doesn't look very "clickable". Is :py:func better?